samri/plotting/create_mesh_featuremaps.py

```diff
@@ -100,6 +100,8 @@ def create_mesh(stat_map,threshold,
 	##TODO: stat_map also possibly already a Nifit1Image, adjust
 	img= nibabel.load(stat_map)
 	img_data = img.get_fdata()
+	if numpy.max(numpy.isinf(img_data)) == True: print("Warning: Infinite values detected; plotting at 0.")
```

Comment:
does `if numpy.max(numpy.isinf(img_data))` not work directly?
Also please put on two lines and indent statement.